spec/lib/sm/client/triage_teams_spec.rb

```diff
@@ -18,7 +18,7 @@
     end
 
     it 'gets a collection of triage team recipients', :vcr do
-      folders = client.get_triage_teams
+      folders = client.get_triage_teams('1234', false)
```

Comment:
Are there specs for when there's already cached data? Should be able to use the classes that use the concern in a before hook to pre-cache.